DigitalOffice/Services/CheckRightsService/Commands/Interfaces/IRemoveRightsFromUserCommand.cs

```
@@ -0,0 +1,18 @@
+﻿using LT.DigitalOffice.CheckRightsService.Models;
+
+namespace LT.DigitalOffice.CheckRightsService.Commands.Interfaces
+{
+    /// <summary>
+    /// Represents interface for a command in command pattern.
+    /// Provides method for removing rights from user.
+    /// </summary>
+    public interface IRemoveRightsFromUserCommand
+    {
+        /// <summary>
+        /// Remove rights from user.
+        /// </summary>
+        /// <param name="request">Request with rights and user id.</param>
+        /// <returns>Return true if successfully else return false.</returns>
+        bool Execute(RemoveRightsFromUserRequest request);
```

no need for bool. just **void**

---

If the user does not have rights to delete rights, then you need to throw an exception?